src/source/feeds.rst

```diff
@@ -1,916 +1,217 @@
 
 .. note::
-   If an ad in the feed remains unchanged (compared to previous import, including image urls), we will skip over this ad and leave
-   it unchanged in our system. This also means we will **not** attempt to download the images and process them again.
-   This is an optimisation that allows us to speed up processing significantly and cut down on calls to your image server.
-   Any change in the ad (including it re-appearing in the feed if it wasn't present the previous time) will update the
-   ad and trigger image re-processing.
-
-======= ====================================================
-Example	.. code-block:: html
-        
-          <ad:vendorId>15839942</ad:vendorId>
-======= ====================================================
-
-.. index:: externalId
-.. _feed_externalId:
-
-externalId
-""""""""""
-.. warning::
-   There is still an **externalId** field in the XSD, this field is replaced by **vendorId**.
-   Please update your XML to reflect this change. This makes naming consistent between feeds and sellside API.
-   The **vendorId** field in the feeds has the same meaning and constraints as the **vendorId** field in the
-   sellside API.
-
-.. index:: sellerName
-.. _feed_sellerName:
-
-sellerName
-""""""""""
-
-Display name of the seller (max. 60 characters).
-
-======= ======================================================
-Example	.. code-block:: html
-        
-         <ad:sellerName>Cups, Caps &amp; Craps</ad:sellerName>
-======= ======================================================
-
-.. index:: title
-.. _feed_t:
+    If you have a dedicated account manager, they can also support with ensuring the URL of your feed is properly connected to your account. 
 
-title
-"""""
+In order to let us download your feed and all the relevant linked content, the access to our outgoing IPs should be whitelisted as described in :ref:`section of the Sellside API<overview_image_downloads>`.
 
-Any string, with minimum and maximum length determined by the category, with a maximum cap of 1024 characters. See :ref:`categories`. URLs are not allowed as part of the title.
 
-======= ====================================================
-Example	.. code-block:: html
-        
-         <ad:title>Goedkope A-merk herenfietsen</ad:title>
-======= ====================================================
-
-.. index:: description
-.. _feed_descr:
-
-description
-"""""""""""
+File Format
+-----------
 
-The description field of the ad.
-Any string, with minimum and maximum length determined by the category. See :ref:`categories`. URLs are not allowed as part of the description.
-All HTML elements except for the ones below will be removed:
+We support feed files in either an XML or TSV format. Feeds are expected to be in UTF-8 encoding. Information on the most actual set of supported data is described on the :ref:`details <feed-details>` page.
```

Comment:
Do you want to mention that the TSV field is google compatible (is it?)

Reply:
Yes, for the new fields, but we still have some ICAS specific fields. It will not disappear, but we want to minimise this set. It sounds like a good idea to write more on that. I have added additional note, and Q/A "What are the new fields."